modules/oneVideoBidAdapter.js

```diff
@@ -244,6 +248,15 @@ function getRequestData(bid, consentData, bidRequest) {
       bidData.imp[0].banner.ext.minduration = bid.params.video.minduration
     }
   }
+
+  if (typeof bid.getFloor === 'function') {
```

Comment:
Not important but we have function type check availability in utils.
in case if you wish to use it. 
`if (utils.isFn(bid.getFloor))`

Reply:
Nice! will switch to that 👍